node/src/chain_spec.rs

```diff
@@ -277,6 +277,19 @@ pub mod mainnet {
 		properties.insert("tokenDecimals".into(), 12.into());
 		properties.insert("ss58Format".into(), 42.into());
 
+		let mut sudo_authority = vec![
+			get_account_id_from_seed::<sr25519::Public>("Charlie"),
+			get_account_id_from_seed::<sr25519::Public>("Dave"),
+			get_account_id_from_seed::<sr25519::Public>("Eve"),
+		];
+		sudo_authority.sort();
+
+		// Define a multisig threshold for 2/3 members
+		let multisig_sudo = pallet_multisig::Pallet::<mainnet_runtime::Runtime>::multi_account_id(
+			&sudo_authority[..],
+			2,
+		);
```

Comment:
nitpick: extract the multisig creation logic in a helper function (to avoid duplication)

Reply:
Yes indeed.On it right away :+1: